.chloggen/jgm-issue-22850.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/influxdb
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: allow empty tag values
```

Comment:
Can we add a regression test here to make sure empty tags are allowed in the future? We can add other test cases as well for the now-allowed invalid characters, non-alphabetical order tag keys, etc.